app/services/stats/r006_kilo_map.rb

```diff
@@ -18,24 +18,25 @@ def self.description
       'Includes a list of all teams and users that respond to requests for information'
     end
 
-
+    # KiloMap isn't really a 'Report' in the normal sense of the word
```

Comment:
In which sense of the word!?

Reply:
Because its just a 'CSV output' thing, and it doesn't use the 'Report' and 'ReportType' structures used by the other reports

Reply:
I've deleted the comment - I agree its not that useful